src/frontend/Felles/Pdf/PdfVisning.tsx

```diff
@@ -40,7 +40,7 @@ const DokumentWrapper = styled.div`
 `;
 
 const PdfVisning: React.FC<PdfVisningProps> = ({ pdfFilInnhold }) => {
-    const [numPages, setNumPages] = useState<number>(0);
+    const [numPages, setNumPages] = useState<number>(1);
     const [pageNumber, setPageNumber] = useState(1);
```

Comment:
Disse to statsene hadde veldig like navn. Nå gjetter jeg litt, men kan den ene hete `totalPageCount` og den andre hete en av følgende: `currentPage`, `currentPageNumber` eller `currentPageIndex`?